app/services/domain/claim.js

```diff
@@ -0,0 +1,47 @@
+const ValidationError = require('../errors/validation-error')
+const FieldValidator = require('../validators/field-validator')
+const FieldsetValidator = require('../validators/fieldset-validator')
+const ErrorHandler = require('../validators/error-handler')
+
+class Claim {
```

Comment:
rather than a generic `Claim` domain object this should be a `FirstTimeClaim`, which needs limited params for constructor